.github/actions/build-downstream/action.yml

```diff
@@ -0,0 +1,105 @@
+name: build
+description: Generates Google/Google-Beta Provider
```

Comment:
It'd be worth rewording this to be a bit clearly. Currently what this generates depends on the input `repo`, and the description implies it could only create TPG or TPGB but you could generate `terraform-google-conversion`.

It could be something like "Generates a given downstream repository from Magic Modules, controlled by the actions inputs"

Reply:
Good point, fixed and should be good for merging!